### PR TITLE
forumspage

### PR DESCRIPTION
**Pull request by ENguyen25**

### CODE REVIEW

File: controllers/api/forum-routes.js [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
-    const dbUserData = await User.findAll();
+    const dbForumData = await Post.findAll();
+
+    const forums = dbForumData.map((forum) => forum.get({ plain: true }));
     
     req.session.save(() => {
       req.session.loggedIn = true;
 
-      res.status(200).render('forum', { dbUserData });
+      // res.status(200).json({dbForumData});
```
Comment (Myuze) [inline controllers/api/forum-routes.js:13]:
```suggestion
```

File: seeds/forumData.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        "date-posted": "September 1, 2021",
+        "author": "Lernantino"
+    },
+    {
+        "title": "mental health issues",
+        "entry": "I can't take it anymore guys, there's just too much going on",
+        "date-posted": "September 1, 2021",
+        "author": "Amiko"
+    }
+]
```
Comment (Myuze) [inline seeds/forumData.json:20]:
```suggestion
]

```

File: views/add-post.handlebars [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+      <br />
+      <textarea id='post_name' name='post_name' rows='2' cols='30'></textarea>
+      <br />
+      <label for='content'>Content:</label>
+      <br />
+      <textarea id='content' name='content' rows='5' cols='30'></textarea>
+      <br />
+      <button type='submit' value='Submit'>Submit</button>
+    </form>
+</class>
```
Comment (Myuze) [inline views/add-post.handlebars:16]:
```suggestion
</class>

```

File: views/forum.handlebars [hunk 4]
```diff
@@ -1 +1,15 @@
...
+          <div class="blog-post-header">
+            <h4>{{title}}</h1>
+          </div>
+      </a>
+</section>
+{{/each}}
+
+<a href="/new-post">
+    <button class="create-new-post">Add Post</button>
+</a>
```
Comment (Myuze) [inline views/forum.handlebars:15]:
```suggestion
</a>

```

**Review by Myuze — CHANGES_REQUESTED** [review]
Couple of fixes, but otherwise looks good.

**Review by Myuze — APPROVED** [review]
Looks good

**Review by LordLowdar — APPROVED** [review]
༼ つ ◕_◕ ༽つ

**Pull request merged**